storage/innobase/xtrabackup/doc/source/backup_scenarios/full_backup.rst

```diff
@@ -185,6 +185,18 @@ option, which will copy the backup to the server's :term:`datadir`:
 If you don't want to save your backup, you can use the :option:`xtrabackup
 --move-back` option which will move the backed up data to the :term:`datadir`.
 
+.. note::
+
+    When ``relay-log-info-repository=TABLE`` is enabled, the instance recovered from the backup has errors in the error log, like the following:
```

Comment:
does it require for 8.0 or only for 5.7?

Reply:
relay-log-info-repository is a variable in 8.0 and 5.7
https://dev.mysql.com/doc/refman/8.0/en/replication-options-replica.html#sysvar_relay_log_info_repository

Reply:
@patrickbirch change LGTM, just place it in the end of the doc - https://github.com/percona/percona-xtrabackup/blame/2.4/storage/innobase/xtrabackup/doc/source/backup_scenarios/full_backup.rst#L216